src/main/java/com/es/phoneshop/model/product/ProductNotFoundException.java

```diff
@@ -0,0 +1,5 @@
+package com.es.phoneshop.model.product;
+
+public class ProductNotFoundException extends Exception{
```

Comment:
Please implement constructor with params, for now you only can use default without any info message.